index.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+require("./src/app.js");
+require("./src/core.js");
+require("./src/window.js");
+
```

Comment:
In the case of the <code>ipc</code> global, we'll need to access it from both the web-universe and the main io.js thread. Does it make sense, then, to instead have each file declare globals it is responsible for?

So, I'm suggesting renaming <code>core.js</code> to <code>ipc.js</code> and have it take care of requiring and assigning the <code>ipc</code> global. The same would be done in <code>app.js</code> for <code>app</code>.

Does that seem reasonable?